File: Day01/baekjoon-1929-re.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(void)
{
    int m, n;
    cin >> m >> n;
    for (int i = m; i <= n; i++)
    {
        if (i == 1)
            continue;
        bool isPrime = true;
        for (int j = 2; j <= sqrt(i); j++)
        {
            if (i % j == 0)
            {
                isPrime = false;
                break;
            }
        }
        if (isPrime)
        {
            cout << i << '\n';
        }
    }
}

File: Day01/baekjoon-1929-re.py
import math
m,n =map(int, input().split())
for i in range(m, n+1):
    if i==1:
        continue
    isPrime = True
    for j in range(2, int(math.sqrt(i))+1):
        if i%j ==0:
            isPrime = False
            break
    if isPrime:
        print(f"{i}")